capstone-frontend/src/components/AdminClubPage.jsx

```diff
@@ -73,20 +74,42 @@ class AdminClubPage extends Component {
   handleMeetingPost = (e) => {
     const history = this.props.history;
     e.preventDefault();
+    const start = moment(new Date(document.getElementById('start-datetime').value));
+    const timezone = Intl.DateTimeFormat().resolvedOptions().timeZone;
+    let rrule = '';
+    if (document.getElementById('None').checked) {
+      rrule = '';
+    } else if (document.getElementById('Daily').checked) {
+      rrule = 'RRULE:FREQ=DAILY;INTERVAL=1'
+    } else if (document.getElementById('Weekly').checked) {
+      const day = start.format('dd');
+      rrule = `RRULE:FREQ=WEEKLY;BYDAY=${day};INTERVAL=1`
+    } else if (document.getElementById('Monthly').checked) {
+      const monthDay = start.format('D');
+      rrule = `RRULE:FREQ=MONTHLY;BYMONTHDAY=${monthDay};INTERVAL=1`
+    } else if (document.getElementById('Yearly').checked) {
+      const month = start.format('M');
+      const monthDay = start.format('D');
+      rrule = `RRULE:FREQ=YEARLY;BYMONTH=${month};BYMONTHDAY=${monthDay};INTERVAL=1`
+    } else {
+      rrule = '';
+    }
```

Comment:
I think this would be a bit better handled by a controlled form (https://reactjs.org/docs/forms.html#controlled-components), but a TODO with an issue is also OK for now.

Reply:
I agree, I could tell there had to be a better way to do this. I'll add a TODO and corresponding issue for now since we're short on time. Thanks for linking the relevant docs.